lib/modules/manager/bazel-module/bazelrc.ts

```diff
@@ -0,0 +1,130 @@
+  readonly entryType = 'import';
+  constructor(readonly path: string, readonly isTry: boolean) {}
+}
+
+export class BazelOption {
+  constructor(readonly name: string, readonly value?: string) {}
+
+  static parse(input: string): BazelOption[] {
+    const options: BazelOption[] = [];
+    const parts = input.split(spaceRegex);
+    for (let i = 0; i < parts.length; i++) {
+      const part = parts[i];
+      if (part.startsWith('--')) {
+        const nameStartIdx = 2;
+        let name: string;
+        let value: string | undefined;
+        // Check for --option_name=option_value
+        const equalSignIdx = part.indexOf('=');
+        if (equalSignIdx >= 0) {
+          name = part.substring(nameStartIdx, equalSignIdx);
+          value = part.substring(equalSignIdx + 1);
+        } else {
+          name = part.substring(nameStartIdx);
+          const nextIdx = i + 1;
+          // Check for --option_name OR --option_name option_value
+          value =
+            nextIdx < parts.length && !parts[nextIdx].startsWith('--')
+              ? parts[nextIdx]
+              : undefined;
+        }
+        const option = new BazelOption(name, value);
+        options.push(option);
+      }
+    }
+    return options;
+  }
+}
+
+export class CommandEntry {
+  readonly entryType = 'command';
+  constructor(
+    readonly command: string,
+    readonly options: BazelOption[],
+    readonly config?: string
+  ) {}
+
+  getOption(name: string): BazelOption | undefined {
+    return this.options.find((bo) => bo.name === name);
+  }
+}
+
+type BazelrcEntries = ImportEntry | CommandEntry;
+
+function shouldProcessLine(line: string): boolean {
+  if (line.length === 0) {
+    return false;
+  }
+  if (line.startsWith('#')) {
+    return false;
+  }
+  return true;
+}
+
+function createEntry(line: string): BazelrcEntries | undefined {
+  const importResult = importRegex.exec(line);
+  if (importResult) {
+    const irGroups = importResult.groups!;
+    return new ImportEntry(irGroups.path, irGroups.type === 'try-import');
+  }
+  const optionResult = optionRegex.exec(line);
+  if (optionResult) {
+    const orGroups = optionResult.groups!;
+    return new CommandEntry(
+      orGroups.command,
+      BazelOption.parse(orGroups.options),
+      orGroups.config
+    );
+  }
+  return undefined;
+}
+
+export function parse(contents: string): BazelrcEntries[] {
+  return contents
+    .split('\n')
+    .map((l) => l.trim())
+    .filter(shouldProcessLine)
+    .map(createEntry)
+    .filter(isNotNullOrUndefined);
+}
+
+async function readFile(
+  file: string,
+  workspaceDir: string
+): Promise<BazelrcEntries[]> {
+  const contents = await fs.readLocalFile(file, 'utf8');
+  if (!contents) {
+    return [];
+  }
+  const entries = parse(contents);
+  const results: BazelrcEntries[] = [];
+  for (const entry of entries) {
+    if (entry.entryType === 'import') {
+      const importFile = entry.path.replace('%workspace%', workspaceDir);
```

Comment:
ensure to remember already read files to not get into infinity recursion. a malicious user could try this or it could be a simple mistake